pkg/domain/domain.go

```diff
@@ -481,23 +489,29 @@ func (*Domain) splitForConcurrentFetch(schemas []*model.DBInfo) [][]*model.DBInf
 	return splitted
 }
 
-func (*Domain) fetchSchemasWithTables(schemas []*model.DBInfo, m *meta.Meta, done chan error) {
+func (*Domain) fetchSchemasWithTables(ctx context.Context, schemas []*model.DBInfo, m *meta.Meta) error {
+	failpoint.Inject("failed-fetch-schemas-with-tables", func(_ failpoint.Value) {
```

Comment:
```suggestion
	failpoint.Inject("failed-fetch-schemas-with-tables", func() {
```

also change the failpoint argument to `1*return()`